package.json

```diff
@@ -28,6 +28,8 @@
     "bcrypt": "0.8.7",
     "bluebird": "3.4.6",
     "body-parser": "1.15.2",
+    "cache-manager": "^2.2.0",
+    "cache-manager-redis": "^0.2.5",
```

Comment:
please use exact version-numbers in package.json. this way we know it won't suddenly stop working